src/common/fragments.py

```diff
@@ -0,0 +1,112 @@
+from pathlib import Path
+from typing import Any, Dict, Optional, Tuple
+
+import pandas as pd
+import streamlit as st
+
+from src.common.utils import get_bar_plot
+
+
+def _parse_config_field(text_input: str, csv_file: Optional[Path], column: str = 'package_name') -> Optional[set]:
```

Comment:
Maybe should we add a default value to csv_file? `csv_file: Optional[Path] = None`

Reply:
Also, I think we should use Optional[str] for the column since if csv_file is None the column should be also None (of course we should handle this case in the function and raise an error if we got incorrect arguments)

Reply:
What behaviour did we expect? Can we send text_input and csv_file at the same time?